.github/CONTRIBUTING.md

```diff
@@ -101,6 +101,52 @@ A good commit may look like this:
 
     [#42] Discover the meaning of life.
 
+### Code Reviews
+
+* Always be constructive when providing feedback (e.g. be explicit on
+  what could improved and how)
+* Use [GitHub suggestion blocks][github-suggest] for fixing typos and
+  small mistakes
+* Positive feedback is as important as negative feedback
+* Only the Pull Request author(s) should push changes to the branch
+  under review
+* Reviewers should be as responsive as possible in providing their
+  reviews.
+* Every review should be completed via either an _Approve_ or a
+  _Request Changes_ action. Not selecting either option should be
+  considered as a _review in progress_.
+* Code reviews should be completed within a reasonable time
+  window. Leaving comments in time-separated _bulks_ is discouraged.
+* It is not acceptable to mark a Pull Request via a _needs work_
+  without further explanation.
+* PR Conversations can be marked as _resolved_ by the PR author, but
+  only if at least one of the following conditions holds:
+    * The issue is addressed via a code change;
+    * the issue will be tackled in a follow up Pull Request (a
```

Comment:
```suggestion
    * The issue will be tackled in a follow up Pull Request (a
```

Reply:
Going for the other way around. After a `;` using a lowercase.